dataset-cellsystems-2021/fish/dataset.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hiPSC-derived cardiomyocyte RNA FISH image dataset",
   "version": "v2021.1",
-  "id": "cellsystems-2021-fish.1",
+  "id": "cellsystems_fish_v2021.1",
```

Comment:
it bothers me that we are storing version and id with version embedded in it but I haven't thought about how to fix that.

Reply:
did you see my proposal in the PR comment? I can make it into a Jira ticket for more discussion too

Reply:
yes, it just wasn't clear if `id` goes away completely then?

Also, I have already named directories and subdirectories with cellsystems-2021-live.1 and cellsystems-2021-fish.1 so several of the download urls are going to still have that.  But since they are fully explicitly specified (a good thing), it won't affect the naming in the dataset json.

(I honestly thought the "2021" was meant to refer to the periodical publication year and not the dataset version).

Reply:
Yeah, I think we would add the id ourselves when we ingest the dataset, so that it's programmatically generated.